clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -288,6 +291,20 @@ private void throwIfOffsetOutOfRange() throws OffsetOutOfRangeException {
     }
 
     /**
+     * If any topic from previous fetchResponse contatains Authorization error, throw ApiException.
+     * @throws ApiException
+     */
+    private void throwIfUnauthorized() throws ApiException {
+        if (!unauthorizedTopicPartition.isEmpty()) {
+            StringBuilder sb = new StringBuilder();
+            for (TopicPartition topicPartition: unauthorizedTopicPartition)
```

Comment:
Minor: space before colon is missing (I know, the fact that this is different for the Java versus Scala code is annoying).